### ISSUE TITLE
Unexpected token .

### ISSUE THREAD

**Issue by sondresorbye** [simeg/gulpfile-generator#18]
```
    .pipe(csslint())
    ^

SyntaxError: Unexpected token .
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:599:28)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at execute (C:\Users\Sondr\AppData\Roaming\npm\node_modules\gulp-cli\lib\versioned\^3.7.0\index.js:28:18)
```

The css task threw an error.

It generated this: 

`gulp.task('css', function() {
    .pipe(csslint())
    .pipe(csslint.reporter())
    .pipe(autoprefixer('last 2 versions'))
    .pipe(gulp.dest(CSS_DEST + '/'))
    .pipe(rename({suffix: '.min'}))
    .pipe(minifycss())
    .pipe(gulp.dest(CSS_DEST + '/'))
});`

**Comment by luizfer** [comment]
Add this before .pipe (csslint ())

`return gulp.src(CSS_SOURCE + '/**/*.css')
    .pipe(plumber({
      errorHandler: function(error) {
        console.log(error.message);
    }}))`

**Comment by simeg** [comment]
I believe this was fixed with #20.
